types/setting.go

```diff
@@ -267,6 +270,24 @@ var (
 		Default:     "3",
 	}
 
+	SettingDefinitionDefaultDataLocality = SettingDefinition{
+		DisplayName: "Default Data Locality",
+		Description: "We say a Longhorn volume has data locality if there is a local replica of the volume on the same node as the pod which is using the volume.\n\n" +
+			"This setting specifies the default data locality mode when a volume is created from the Longhorn UI. For Kubernetes configuration, update the `dataLocality` in the StorageClass\n\n" +
+			"The available data locality modes are: \n\n" +
+			"- **disable** is the default mode. There may be a local replica of the volume on the same node as the consuming pod or there may be not. Longhorn doesn't do anything\n" +
```

Comment:
I think it's `may not be` instead of `may be not` ?

Reply:
Thank you! Fixed